src/server/views/index.pug

```diff
@@ -60,6 +60,55 @@ block content
     //-   p Learn the key concepts that run our interconnected world from iPhones through Instagram to traffic lights.
 
   .container-fluid#showcase
+    .row
+      a(class="wrapper-link",href= env.EDITOR_ADDRESS+'/?action=example&ProjectName=Weather', target='_blank') 
+        .element-item.col-xs-12.col-sm-4.col-md-3
+          .h-thumbnail
+            .img-container
+              img.img-responsive.center-block.img-thumbnail(src= `${env.EDITOR_ADDRESS}/api/examples/Weather/thumbnail?aspectRatio=1.33333`, style='width: 100%;')
+              .overlay
+                .caption.text-center Click to green flag to get a map. Use the +/- and arrow keys zoom and navigate. Click anywhere to get the current conditions.
+            ul.list-group
+              //- span.label.label-success SERVICES
+              li.list-group-item.title
+                b Weather
+                //- span.pull-right Roles: NUMBEROFROLES
+      a(class="wrapper-link",href= env.EDITOR_ADDRESS+'/?action=example&ProjectName=Battleship&editMode=true', target='_blank') 
+        .element-item.col-xs-12.col-sm-4.col-md-3
+          .h-thumbnail
+            .img-container
+              img.img-responsive.center-block.img-thumbnail(src= `${env.EDITOR_ADDRESS}/api/examples/Battleship/thumbnail?aspectRatio=1.33333`, style='width: 100%;')
+              .overlay
+                .caption.text-center Select the Room tab and click the "player 2 role" to invite a friend to play. Then click the green flag. Once you placed your ships, hit space to start.
+            ul.list-group
+              //- span.label.label-success SERVICES
+              li.list-group-item.title
+                b Battleship
+                    i.fa.fa-users.pull-right(title="Multiplayer")
```

Comment:
This might look better if you add `fa-lg` to it (since the whole card is larger)